prov/rxm/src/rxm_cq.c

```diff
@@ -518,6 +518,8 @@ static ssize_t rxm_lmt_send_ack_fast(struct rxm_rx_buf *rx_buf)
 
 	assert(rx_buf->conn);
 
+	RXM_LOG_STATE(FI_LOG_CQ, rx_buf->pkt, RXM_LMT_READ, RXM_LMT_ACK_SENT);
```

Comment:
We should also set the state to RXM_LMT_ACK_SENT.

Reply:
Yes, you're right. But it isn't needed, because this function calls `rxm_finish_send_lmt_ack()` and set state to `RXM_LMT_FINISH`.